vmm/task/Cargo.toml

```diff
@@ -17,7 +17,6 @@ serde = { version = "1.0.133", features = ["derive"] }
 serde_json = "1.0.74"
 oci-spec = "0.5.4"
 crossbeam = "0.8.1"
-env_logger = "0.9.0"
```

Comment:
log crate is no longer depended?

Reply:
Initializing crate env_logger and crate tracing at the same time gives a runtime error. So I think it is not used anymore.

Reply:
OK, thus shoule the log crate in Cargo.toml be removed?

Reply:
Ah, Maybe we can still use log crate as facade and configure tracing as the log backend implementation. Let me take a look.

Reply:
I saw the `log` in `common/Cargo.toml` was removed in this commit

Reply:
I will take it back later.